lib/fe_argv.mli

```diff
@@ -0,0 +1,54 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+(** This module supports building a large command line (argument) vector
+ * incrementally in a monad.
+ *)
+
+type argv
+type 'a t      (** monad, holding current command line *)
+
+(** basic functions to construct a [t] value. More below *)
+val return: 'a -> 'a t
+val bind:   'a t -> ('a -> 'b t) -> 'b t
```

Comment:
Why you don't export also `>>=` in the `mli`?

Reply:
I could but I want to avoid that clients routinely open the module. I think a ``let (>>+) = Fe_argv.bind` is easy enough.